src/utils/api.js

```diff
@@ -14,10 +15,10 @@ const api = {
   },
 
   containers: {
-    get: id => get(`/containers/${id}`),
-    list: () => get("/containers/list_root"),
-    listParents: id => get(`/containers/${id}/list_parents/`),
-    listChildren: id => get(`/containers/${id}/list_children/`),
+    get: id => get(`/containers/${id}`, nested),
+    list: (page = {}) => get(`/containers/`, {...page, ...nested}),
```

Comment:
Why is `/containers/list_root` changed for `/containers/`? IIRC, this would have a few implications.

Reply:
This was done so we would see all containers not just the connected ones...

Reply:
What do you mean by not just the connected ones?